### DISCUSSION TITLE
Unifying endpoints from the chainkit and neutrino sub servers

### DISCUSSION THREAD

**Discussion by ErikEk** [lightningnetwork/lnd#7307] [General]
I've noticed that the chainkit sub server have been merged into the 0.16 branch. It would be good to unify the responses from the neutrino and chainkit sub servers, or depreciate some commands in the neutrino sub server and move them into the chainkit sub server.  This involves endpoints such as getblock, getblockheader, getblockhash etc...
It would be good if this could be done before the chainkit sub server is merged into a stable major release imo.
I don't mind submitting a pr.

**Comment by yyforyongyu** [comment]
Good point, and there's already a PR #7284 for it.

**Reply by ErikEk** [reply]
@yyforyongyu Great.
